Frameworks/UIKit/StarboardXaml/StarboardXaml.cpp

```diff
@@ -374,6 +374,28 @@ void UIApplicationInitialize(const wchar_t* principalClassName, const wchar_t* d
     _ApplicationLaunch(ActivationTypeLibrary, nullptr);
 }
 
+// This is the initialization function for functional tests
+UIKIT_EXPORT
+void UIApplicationInitializeFunctionalTest(const wchar_t* principalClassName, const wchar_t* delegateClassName) {
```

Comment:
That doesn't go down the same codepath.

---
In reply to: [96533713](https://github.com/Microsoft/WinObjC/pull/1716#discussion_r96533713) [](ancestors = 96533713)